src/marqo/client.py

```diff
@@ -9,6 +9,11 @@
 from marqo._httprequests import HttpRequests
 from marqo import utils, enums
 from marqo import errors
+from marqo.version import __minimum_supported_marqo_version__
```

Comment:
Let's add a method `marqo.version.minimum_supported_marqo_version()` to access this value. Direct access to private fields is bad practice

Reply:
thanks! resolved